modules/table/src/main/java/org/apache/ignite/internal/table/InternalTable.java

```diff
@@ -374,6 +425,13 @@ Publisher<BinaryRow> lookup(
      */
     List<String> assignments();
 
+    /**
+     * Gets a list of current table leader assignments with raft term.
+     *
+     * @return List of current leader assignments with raft term.
+     */
+    List<IgniteBiTuple<String, Long>> leaderAssignmentsWithTerm();
```

Comment:
Also, you have to understand Placement driver is not a static collection. It will be a service with a method to get a PrimaryReplica by ReplicationGroupId.

Reply:
Done, thanks. 
Created `PrimaryReplica` in `table` module.